platformsh-flex-env.php

```diff
@@ -96,13 +96,14 @@ function mapPlatformShDatabase() : string
                     $endpoint['port'],
                     $endpoint['path']
                 );
-                
-                $guessedVersion = strpos($endpoint['type'], ":");
 
                 switch ($endpoint['scheme']) {
                     case 'mysql':
+                        $type = $endpoint['type'] ?? 'mysql:10.2';
```

Comment:
I guess `mysql:10.2` is the default when none is provided? Also `mysql:10` sound more logical. What about making that explicit via a constant?

```suggestion
                        const DEFAULT_MYSQL_ENDPOINT_TYPE = 'mysql:10.2';
                        $type = $endpoint['type'] ?? DEFAULT_MYSQL_ENDPOINT_TYPE;
```

Reply:
Making it a constant makes sense, but in that case it should not be in the function but defined in the top of the file.

Which probably means namespacing the file to avoid possible collisions with the application.  Bah.

Reply:
Also there is no "mysql:10" container image, only 10.0, 10.1, and 10.2.